examples/occupancy/ext_estimatedTimetable_response.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- (C) Copyright 2005-2012 CEN SIRI -->
+<Siri xmlns="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0" xsi:schemaLocation="http://www.siri.org.uk/siri ../../xsd/siri.xsd">
+	<ServiceDelivery>
+		<ResponseTimestamp>2023-11-16T09:30:47-05:00</ResponseTimestamp>
+		<Status>true</Status>
+		<EstimatedTimetableDelivery version="2.0">
+			<ResponseTimestamp>2023-11-16T09:30:47-05:00</ResponseTimestamp>
+			<ValidUntil>2023-11-17T09:30:47-05:00</ValidUntil>
+			<EstimatedJourneyVersionFrame>
+				<RecordedAtTime>2023-11-16T09:30:47-05:00</RecordedAtTime>
+				<!--==journey 1========================== -->
+				<EstimatedVehicleJourney>
+					<LineRef>unknown</LineRef>
```

Comment:
Bad example?

Reply:
why bad?

Reply:
Why would you _not_ know the LineRef in any decent system?

Reply:
I see. I can add it here. but in that case: We don't have a decent system currently :-)